product-specifications/example-3.md

```diff
@@ -0,0 +1,47 @@
+# Project Specification: 'Maze Runner' CLI Game 🏃
+
+## Overview
+
+'Maze Runner' is a text-based, Command-Line Interface (CLI) game designed for users who appreciate the simplicity and challenge of classic text-based games. The main objective is to navigate a maze and find the exit as quickly as possible.
+
+## Goals
+
+1. To provide an engaging gaming experience to users who enjoy text-based games.
+2. To challenge players with randomly generated mazes of varying complexity.
+3. To include a scoring system based on maze complexity and completion time.
+
```

Comment:
can we allow the user to chose their player

Reply:
Great idea, I think having different players with different abilities would be cool. Let's plan to add that in V2